packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -101,6 +103,7 @@ Tasks.defaultProps = {
   tasks: []
 }
 
+@withTheme
```

Comment:
Quick suggested change here, from [the docs](https://github.com/mobxjs/mobx-react#provider-and-inject) on `inject` and `observer`:

> When using both `@inject` and `@observer`, make sure to apply them in the correct order: observer should be the inner decorator, inject the outer. There might be additional decorators in between.

So, this should be:
```
@inject(storeMapper)
@withTheme
@observer
```

Reply:
Sounds good. I copied the order here from another component, so it might need to be fixed somewhere else.

Reply:
The Field Guide was applying the decorators in this order, so I've fixed that here.